torchgeo/trainers/segmentation.py

```diff
@@ -180,33 +196,159 @@ def configure_losses(self) -> None:
     def configure_metrics(self) -> None:
         """Initialize the performance metrics.
 
-        * :class:`~torchmetrics.Accuracy`: Overall accuracy
-          (OA) using 'micro' averaging. The number of true positives divided by the
-          dataset size. Higher values are better.
-        * :class:`~torchmetrics.JaccardIndex`: Intersection
-          over union (IoU). Uses 'micro' averaging. Higher valuers are better.
+        * :class:`~torchmetrics.classification.MulticlassAccuracy`: Overall accuracy
```

Comment:
This change seems like going in the wrong direction, shouldn't we keep the generic class links?